lib_spi/src/spi_sync.xc

```diff
@@ -272,6 +272,10 @@ void spi_master(server interface spi_master_if i[num_clients],
                 get_mode_bits(mode, cpol, cpha);
 
                 //xassert(device_index < num_slaves);
+                if (!isnull(cb)){
+                    set_port_use_on(sclk);
```

Comment:
Didn't we also decide to call set_port_use_off() after spi transaction was complete?

Reply:
Thanks for the review. We did agree that it would be good to leave the port in an off state in between begin and end calls. Later I realised that turning port off makes the pin high impedance, and thought that a floating pin is worse than a port that's always on.